etc/greenboot/green.d/50_greenboot_motd.sh

```diff
@@ -1,4 +1,16 @@
 #!/bin/bash
 set -euo pipefail
 
-ln -snf /etc/greenboot/motd/greenboot.motd /run/greenboot/motd
+if [ ! -d /etc/motd.d ]; then
+    mkdir /etc/motd.d
+fi
+
+greenboot_logs=`journalctl -u greenboot.service -p 0..5 -b -0 -o cat`
```

Comment:
nit: I think generally `$(command arg)` is preferred over `'command arg'` (using backticks)